doc/en/reference/plugin_list.rst

```diff
@@ -47,7 +47,7 @@ This list contains 1411 plugins.
    :pypi:`pytest-adf-azure-identity`                Pytest plugin for writing Azure Data Factory integration tests                                                                                                                                                                                                                                                                                                                          Mar 06, 2021    4 - Beta               pytest (>=3.5.0)
    :pypi:`pytest-ads-testplan`                      Azure DevOps Test Case reporting for pytest tests                                                                                                                                                                                                                                                                                                                                       Sep 15, 2022    N/A                    N/A
    :pypi:`pytest-affected`                                                                                                                                                                                                                                                                                                                                                                                                                  Nov 06, 2023    N/A                    N/A
-   :pypi:`pytest-agent`                             Service that exposes a REST API that can be used to interract remotely with Pytest. It is shipped with a dashboard that enables running tests in a more convenient way.                                                                                                                                                                                                                 Nov 25, 2021    N/A                    N/A
+   :pypi:`pytest-agent`                             Service that exposes a REST API that can be used to interact remotely with Pytest. It is shipped with a dashboard that enables running tests in a more convenient way.                                                                                                                                                                                                                 Nov 25, 2021    N/A                    N/A
```

Comment:
I couldn't tell if this is a list of descriptions that come from the modules, in which case the fixes would belong in the modules -- but searches didn't seem to show lots of indications where I'd expect if that was the case, so...